pkgs/applications/video/mpv/default.nix

```
@@ -19,6 +19,11 @@
   , libcdio          ? null
   , libcdio-paranoia ? null
 
+, vulkanSupport ? true
```

```suggestion
, vulkanSupport ? stdenv.isLinux
```
Let's not brake darwin build.

---

I was not aware this was gonna break the darwin build. Fixed :)